packages/terra-time-input/translations/en-GB.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
What is the reason for deleting some of these translations files (nl-BR; pt-BR, fr_FR, etc)?

Reply:
There were no differences in those files from their parent locale - only dual maintenance. For example _fr_FR_ (French, France) and _fr_ (French) were identical. End-users in fr_FR locale will receive the fr localization at runtime. We should add these back only if one requires different L10n, such as some of the Spanish locales have.

Reply:
The pattern we have been using has always been including the translations files for all supported locales (even if the strings for the locale are identical to the language. I think our [aggregate-translations package](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-aggregate-translations/src/aggregate-messages.js#L11) will warn if we have missing locale files. 

This would also introduces inconsistency with other existing packages. 'm not totally against removing translations files that have no difference with the base language file. If we do go this route and to minimize inconsistencies, I think we should make an effort to do this across all packages, potentially update our generators, update the logic in aggregate-translations, potentially update workflow-server has it currently has a validation script that would fail if there are missing strings that are not translated for a given locale.

So there is quite a bit of work that is needed to get done in harmony if we want to get this done right.

Given this, I personally think we should keep these files for now until we have a better plan in place to remove the matching locales.

Reply:
@benbcai actually there was already a change made in aggregate translations to not warn about that any more: https://github.com/cerner/terra-toolkit/commit/36de8d91d5905f5aa25659749299113168233376
And workflow-framework only checks that there are translations for the base locales, not regional translations:
https://github.cerner.com/orion/workflow_server/blob/master/client/tests/jest/validate-translations.test.js#L224-L231

Reply:
I don't think we need to necessarily make intentional changes to remove the extraneous translations in all our components, but when the translations team tells us that there's no difference between a regional locale and the base locale, there's not any need to actually include them, since aggregate-translations will automatically fallback, it reduces the chance of copy/paste errors. I don't see any harm in cleaning them up if we happen to be updating translations anyway.

Reply:
I verified that our L10n falls back correctly. I visited the deployment, flipped the locale to fr_FR and saw the correct fr fallbacks. Given that and the comments above, I think this is fine. I agree no need to rip out translations right away. It can be done over time as we touch projects.

![image](https://user-images.githubusercontent.com/52507228/153653657-f353066d-a5ad-4e23-90ed-e836f2896591.png)